src/framework-configuration.js

```diff
@@ -146,8 +176,7 @@ export class FrameworkConfiguration {
    * @return Returns the current FrameworkConfiguration instance.
   */
   feature(plugin: string, config?: any): FrameworkConfiguration {
-    plugin = plugin.endsWith('.js') || plugin.endsWith('.ts') ? plugin.substring(0, plugin.length - 3) : plugin;
```

Comment:
Yes. As I recall, at some point in the past, this was an issue. It might be when in-browser compilation was happening. You may need to also test our beginner kits, which don't use gulp but do everything in browser. In particular, I'm not sure how the TS kit will work. It currently uses a plugin to enable TS. I'm wondering if that can be removed with the latest jspm/system.js